docs/reference/pf-command-reference.md

```diff
@@ -905,7 +905,8 @@ pf service start [--port]
 ```
 
 #### Examples
-Start prompt flow service.
+Start prompt flow service. We endeavor to start the service on the default port 23333. And if it's already taken, we will sequentially probe new ports, incrementing by one each time.
```

Comment:
document here and also the pf service start --help that will remember and use the last started port if port not set

Reply:
we will not use the last started port for now, but always start from default port or usered passed port.

when start service: always start from 23333

when stop service: will clear port info

Reply:
offline synced and updated